UK-LON-X1

oli-barker opened the pull request.

oli-barker commented:
Setting up new building please ignore

charbull commented:
Hi @oli-barker can you please sign the cla

Thank you

charbull commented on ontology/yaml/resources/fields/telemetry_fields.yaml:
```diff
@@ -772,3 +772,10 @@ literals:
 - mixed_air_dewpoint_temperature_sensor
+
+
+# Added for UK-LON-X1
```
can you please remove this tag (UK-LON-X1)

oli-barker commented:
> @googlebot I signed it

oli-barker commented:
> @googlebot I signed it!

tasodorff commented on ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:
```diff
@@ -18,6 +18,17 @@
 ##############################################################
 
+
+TRU:
```
Doesn't match BOS v2.3 standard. Fix this.

tasodorff commented on ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:
```diff
@@ -18,6 +18,17 @@
 
+
+TRU:
+  id: "oli0001"
```
Omit all IDs, the system will append these if needed.

tasodorff commented on ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:
```diff
@@ -18,6 +18,17 @@
+
+TRU:
+  id: "oli0001"
+  description: "Tag for trench heating units. Trench Heating units are in floor airconditioning units. 
```
Clarify "particular to under-floor air distribution systems" or something similar.

tasodorff commented on ontology/yaml/resources/HVAC/entity_types/TRU.yaml:
```diff
@@ -0,0 +1,30 @@
+
+# This file defines Trench heating unit types.
+
+TRH_DFSS_DFVSC_DSP_CHWZC:
```
This name doesn't match the types implemented below. These should match by convention.

tasodorff commented on ontology/yaml/resources/HVAC/entity_types/TRU.yaml:
```diff
@@ -0,0 +1,30 @@
+  description: "Typical trench heater with cooling-only"
+  is_canonical: true
+  implements:
+  - TRU
```
fix

tasodorff commented on ontology/yaml/resources/HVAC/entity_types/TRU.yaml:
```diff
@@ -0,0 +1,30 @@
+  is_canonical: true
+  implements:
+  - TRU
+  - DFVSC
```
Please also implement DFSS

tasodorff commented on ontology/yaml/resources/HVAC/entity_types/TRU.yaml:
```diff
@@ -0,0 +1,30 @@
+  is_canonical: true
+  implements:
+  - TRU
+  - DFVSC
```
Same note as above; please implement DFSS also

tasodorff commented on ontology/yaml/resources/HVAC/entity_types/TRU.yaml:
```diff
@@ -0,0 +1,30 @@
+  - CHWZTC
+  - RMM
+
+TRH_DFSS_DFVSC_DSP_HWZC_CHWZC:
```
Same note as above; match implemented types.

tasodorff commented on ontology/yaml/resources/METERS/entity_types/HM.yaml:
```diff
@@ -24,4 +24,17 @@ HM_CHW:
+  - cooling_thermal_energy_accumulator
+
+HM_HWS:  
+  id: "oli0004"
```
reminder to remove all fake IDs

tasodorff reviewed (changes requested):
Main comment is to change trench heater abbreviation to match BOS standard. Several minor fixes (remove all IDs, add a few requested abstract types, type name should match concatenated 'implements' types by convention, etc.).

tasodorff commented:
I still don't see the type names updated in your latest PR.

oli-barker commented:
> I still don't see the type names updated in your latest PR.

I updated the TRU entry in general types to TRH in line with BOS 2.3. Also changed the TRU.yaml file to TRH.yaml and defined 2 types of trench heater. One with just heating and one with both. I haven't defined TRHC as its effectively a subtype of the main item. This seemed the same in line with FCU where it can be any configuration of heating and cooling etc but is still an FCU.

tasodorff commented on ontology/yaml/resources/HVAC/entity_types/AHU.yaml:
```diff
@@ -3566,4 +3566,52 @@ AHU_US_SVL_MAT2_4:
+  - supply_air_flowrate_sensor
+  
+  
+AHU_UK_LON_X1_1:
```
A few notes:
1. Why dont we have any information for the supply/exhaust fans (command/status/speeds), orr any outside air information?
2. Do not use the name of the site in the type name.

tasodorff commented on ontology/yaml/resources/HVAC/entity_types/AHU.yaml:
```diff
@@ -3566,4 +3566,52 @@ AHU_US_SVL_MAT2_4:
+  - supply_air_flowrate_sensor
+  - return_air_temperature_sensor  
+  
+AHU_UK_LON_X1_2:
```
Same note as above -- remove Google specific name.

tasodorff commented on ontology/yaml/resources/fields/telemetry_fields.yaml:
```diff
@@ -772,3 +772,38 @@ literals:
+
+#added for UK-LON-X1
+
+- heating_wheel_percentage_sensor
```
heating wheel what percentage sensor? 

percent of air flowing through it? speed at which its rotating? percent capacity? needs more specificity.

tasodorff commented on ontology/yaml/resources/fields/telemetry_fields.yaml:
```diff
@@ -772,3 +772,38 @@ literals:
+- discharge_air_filter_differential_pressure_setpoint
+- return_air_filter_differential_pressure_sensor
+- return_air_filter_differential_pressure_setpoint
+- cooling_coil_percentage_command
```
is this the valve?

tasodorff commented on ontology/yaml/resources/fields/telemetry_fields.yaml:
```diff
@@ -772,3 +772,38 @@ literals:
+- cooling_coil_status:
+  - ACTIVE
+  - INACTIVE
+- heating_coil_percentage_command
```
why are you using this and not heating_water_valve_percentage command? is it not hydronic? if not, what is it?

tasodorff commented on ontology/yaml/resources/fields/telemetry_fields.yaml:
```diff
@@ -772,3 +772,38 @@ literals:
+  - ACTIVE
+  - INACTIVE
+- heating_coil_percentage_command
+- frost_coil_status:
```
again, what is this a status of? flow through coil?

tasodorff commented on ontology/yaml/resources/fields/telemetry_fields.yaml:
```diff
@@ -772,3 +772,38 @@ literals:
+  - ACTIVE
+  - INACTIVE
+- max_supply_air_temperature_setpoint
+- min_supply_air_temperature_setpoint
```
same as above; use `low_limit` if not an aggregation.

tasodorff reviewed (changes requested):
Clarifications needed on some definitions, and alternate names required.

tasodorff commented:
@oli-barker what is the status of these? please make changes or the PR will be closed without committing the changes.

tasodorff commented:
@oli-barker status?

tasodorff commented:
Closed due to inactivity. Can reopen if necessary.
